python/lsst/metaserv/schemaToMeta.py

```diff
@@ -47,7 +47,7 @@ class SchemaToMeta(object):
 
     _tableStart = re.compile(r'CREATE TABLE (\w+)*')
     _tableEnd = re.compile(r"\)")
-    _engineLine = re.compile(r'\) (ENGINE|TYPE)=(\w+)*;')
+    _engineLine = re.compile(r'\)[\s]*(ENGINE|TYPE)[\s]*=[\s]*(\w+)*;')
     _columnLine = re.compile(r'[\s]+(\w+) ([\w\(\)]+)')
```

Comment:
Remove brackets around `\s`.  The backslashes before the parentheses in the character class (within brackets) are not needed and make it hard to read.
You might want to allow the column definition to start at the beginning of the line by using `\s*`.
This regexp requires a single space, but it also matches a lot more than just a column name and type (for example, `id i)n(t` matches).  You might do better with `(\w+\s*(\([\d\s,]+\))?)` for the column type part if you really wanted just column definitions, except that I don't think you even care about that.
In sum, what you really want here is `r'\s*(\w+)\s+\w+'`.